cmd/kubeapps-apis/plugins/kapp_controller/packages/v1alpha1/server_ctrl_packages.go

```diff
@@ -524,8 +524,7 @@ func (s *Server) GetInstalledPackageDetail(ctx context.Context, request *corev1.
 		}
 	}
 	// trim the new doc separator in the last element
-	//nolint:staticcheck
-	valuesApplied = strings.Trim(valuesApplied, "---")
+	valuesApplied = strings.TrimSuffix(valuesApplied, "---")
```

Comment:
The golint gh action said we were using `.Trim` incorrectly: it trims chars (unicode runes), so using `---` was -partially- wrong, as it wasn't removing the occurrences of the string `---`  but rather removing every occurrence of `-`.

To trim strings, we should use TrimSuffix or TrimPrefix instead (or Replace even). See https://topic.alibabacloud.com/a/the-difference-between-the-trimright-and-trimsuffix-of-the-golang-strings-package_1_38_30927804.html

Squeezing this change in this PR.